app/assets/stylesheets/language.scss

```diff
@@ -101,3 +103,14 @@
     background-color: #f5f5f5;
   }
 }
+
+@media (max-width: 400px) {
```

Comment:
Would prefer if you used Bootstraps [media queries](http://getbootstrap.com/css/#grid-media-queries) rather than setting your own custom breakpoints to keep everything nice and clean!